packages/playwright/package.json

```diff
@@ -54,7 +54,7 @@
     "axe-core": "~4.9.0"
   },
   "devDependencies": {
-    "@playwright/test": "^1.34.3",
+    "@playwright/test": "^1.40.1",
```

Comment:
playwright@1.41 introduced a regression in how they handled lazy loaded (unloaded) iframes. The tests were working until 1.41 and now they are broken. Downgrading for now while we work on a solution for playwright that is backwards compatible.